### ISSUE TITLE
7z command not found on Mac

### ISSUE THREAD

**Issue by jjacobs33589** [kolbasa/cordova-plugin-apkupdater#8]
I'm trying to use this plugin on a mac. When I run the following command

node create-manifest.js <version> <chunk-size> <apk-path> <output-path>

I get this error

Compressing Update
Error: Command failed: 7z a -v100k -mm=Deflate -mfb=258 -mpass=15 -r /update/update.zip /application.apk
/bin/sh: 7z: command not found

**Comment by kolbasa** [comment]
There are several ways to install 7zip on MacOS.

I personally use [Homebrew](https://brew.sh/index_de):
```bash
brew update
brew install p7zip
```

Alternatively with [MacPorts](https://www.macports.org/):
```bash
sudo port install p7zip
```

I didn't think about that. I will add the info to the documentation.

**Comment by jjacobs33589** [comment]
Thank you that works. 

However, when I run the following command 
`node create-manifest.js 1.0.0 1m /home/user/app.apk /home/user/update`

I have an apk that is 6.3MB but the folder with the compressed files is 28MB. This is the manifest.json output of the above command.

`{"version":"1.0.0","chunks":["c7bab217d425ff92b6bcfb0de8ea13b3","f5df681a571f1cccf9e901c2917e64e5","cc2608def66ff8ad43b18a42ec6d8fee","f85b5920f4d6fbfdd2d9fb40023c2ac2","5f23a612bdda42d658eea501c82940de","cea626663b481a98e8219bfd174434b8","6a6460ee18360d76d452c213e13004c7","54e37dfb18a8d9aaa1d7c18f57d28326","c61231bc6e11447e337493875fa767de","df378115111aecfab48360ecf7633557","8fd8dd6e84349d284a386cdb0b9b99b4","fd18cde3c82380525784600bdc721a41","77acbb967b12d575bfcffa3e6640d044","ac5c3f63ff2bd79581b5fd48216dc038","71a50da26f69a94b37f33106a44c74e8","00d2d010764012c886c4c44bc5f936a0","33ff1d7d31866c3f444eb256ed618cbe","bcf483d7c6539225b7b2f8b75fe1a843","f5f14168416d067e748aadb49cc106b8","c5736b2719aa19aba3edf7c1209ce19d","752dbf1ba72060752e7ab009dd206c61","eaeaa9b394ac03bcbcb1921fafad909f","f86147e515dc7eea0ce9c1b732616092","69c6699eaff3baa3cfff46753cb84f0e","05046116402e6e9dbe9a0990f96a56e2","d722aae30da041c85e60b3521f3458cf","e49d2eaf08cb7897c035734747270702"],"compressedSize":28181101,"size":6274969,"sum":"932ad797b5434eeb68ff814419a3cfca"}`

It generated 27 zipped files that are each 1M

**Comment by kolbasa** [comment]
# :satisfied: 
WTF, how?

I just tried again with this [GMail APK](https://www.apkmirror.com/apk/google-inc/gmail/gmail-2020-07-12-323583380-release-release/gmail-2020-07-12-323583380-release-android-apk-download/).

```bash
node create-manifest.js 1.0.0 10m /home/user/gmail.apk /home/user/update
```
The file is compressed correctly from `59.9MB` to `40MB`.
```js
{
    'version': '1.0.0',
    'chunks': [
        '37cd5fa99bc48bd25b3471c90022058a',
        '7baf365ed88a7258809a83107fb61096',
        '1bb90e1df20c4c3ee3230c09f4e33675',
        'eeaa715f247faaea7c1a26c722e0d092',
        '7c1be9c464fa2eca91087323329a903e'
    ],
    'compressedSize': 41948748,
    'size': 62785275,
    'sum': 'fb1d2f29a8d2c929e142e96fe2f9e1a7'
};
```

Which 7Zip version do you have?
Maybe you can send me the file, if that's possible. My email address is: michael.jedich@pm.me.
Otherwise, I have no idea how this can be. 

The script actually does nothing else but compress the file with 7zip:
```bash
7z a -v1m -mm=Deflate -mfb=258 -mpass=15 -r /home/user/update/update.zip /home/user/app.apk
```

**Comment by kolbasa** [comment]
OK, I found the mistake.
![image](https://user-images.githubusercontent.com/3901582/89456370-0fccc180-d764-11ea-83f5-778b4875c12e.png)
This also searched for the `app.apk' in subfolders. And you probably had some folders with other versions of this file.

**Comment by jjacobs33589** [comment]
Thank you, I've done some testing too and was going to say the same thing. I did have other folders in there

**Comment by kolbasa** [comment]
Fixed in version `1.2.1`. Thanks for the feedback. There's no way I could have found this on my own.
If there's anything else, let me know. I'll close this issue. :heart:
